File: CI.c
#include <stdio.h>
#include <math.h>

int main() {
    float P, R, T, A, CI;

    printf("Enter Principal Amount: ");
    scanf("%f", &P);

    printf("Enter Rate of Interest: ");
    scanf("%f", &R);

    printf("Enter Time (in years): ");
    scanf("%f", &T);

    A = P * pow((1 + R/100), T);
    CI = A - P;

    printf("Compound Interest = %.2f", CI);

    return 0;
}

File: CI.java
import java.util.Scanner;

public class CompoundInterest {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter Principal Amount: ");
        double P = sc.nextDouble();

        System.out.print("Enter Rate of Interest: ");
        double R = sc.nextDouble();

        System.out.print("Enter Time (in years): ");
        double T = sc.nextDouble();

        double A = P * Math.pow((1 + R/100), T);
        double CI = A - P;

        System.out.println("Compound Interest = " + CI);

        sc.close();
    }
}
